app/components/contestantRoundList.tsx

```diff
@@ -0,0 +1,39 @@
+import Round from './round'
+import Team from '../models/Team'
+
+export default function ContestantRoundList({
+        perfectRoundScores,
+        contestantRoundScores,
+        perfectTeamList,
+        contestantTeamList
+    }: {
+        perfectRoundScores: number[]
+        contestantRoundScores: number[]
+        perfectTeamList: Team[]
+        contestantTeamList: Team[]
+    }) {
+
+    let grandTotal = 0
+    let contestantGrandTotal = 0
+
+    return (
+        <div className="text-center">
+            {perfectRoundScores.map((score, roundNumber) => {
+                grandTotal += score
+                let contestantRoundScore = contestantRoundScores[roundNumber]
+                contestantGrandTotal += contestantRoundScore
+
+                return <Round
+                    key={"round"+roundNumber}
+                    roundNumber={roundNumber}
+                    perfectTeamList={perfectTeamList}
+                    contestantTeamList={contestantTeamList}
+                    perfectWeekScore={score}
```

Comment:
```suggestion
                    perfectWeekScore={perfectScore}
```

Purely a suggestion but score certainly works here.

Reply:
I'm not going to accept this specific change only because it should be made intandum with renaming/declaring the variable so I will add a whole other comit.